main.tf

```diff
@@ -110,6 +117,9 @@ resource "aws_eks_cluster" "default" {
 
   depends_on = [
     aws_iam_role_policy_attachment.amazon_eks_cluster_policy,
-    aws_iam_role_policy_attachment.amazon_eks_service_policy
+    aws_iam_role_policy_attachment.amazon_eks_service_policy,
+    aws_cloudwatch_log_group.default
```

Comment:
will this `depend_on` work if `count = 0` for "aws_cloudwatch_log_group" "default"  (it's not provisioned)?

Reply:
@kskewes can you confirm that  depend_on `aws_cloudwatch_log_group.default` will work if `aws_cloudwatch_log_group.default` is disabled?

Reply:
Will validate soonest, please stand by.